topics/assembly/tutorials/vgp_workflow_training/tutorial.md

```diff
@@ -166,61 +243,139 @@ Once we have imported the datasets and the workflows, we can start with the geno
+3. In the **Workflow: VGP genome profile analysis** menu:
+   - {% icon param-collection %} "*Collection of Pacbio Data*": `7: HiFi_collection`
+   - "*K-mer length*": `31`
+   - "*Ploidy*": `2`
+4. Click on the <kbd>Run workflow</kbd> buttom
+
+This should like this:
+
+------
+
+![Figure 5: Parameters of *k*-mer profiling workflow](../../images/vgp_assembly/wf1_launch_ui.png  "Workflow main menu. The workflow menu lists all the workflows that have been imported. It provides useful information for organizing the workflows, such as last update and the tags. The worklows can be run by clicking in the play icon, marked in red in the image.")
+
+------
+
+> <comment-title>K-mer length</comment-title>
+> In this tutorial, we are using a *k*-mer length of 31. This can vary, but the VGP pipeline tends to use a *k*-mer length of 21, which tends to work well for most mammalian-size genomes. There is more discussion about *k*-mer length trade-offs in the extended VGP pipeline tutorial.
+{: .comment}
+
+### **Step 3**: Refill your coffee
+
+Assembly is not exactly an instantaneous type of analysis - this workflow will take approx 15 minutes to complete.
+
+### **Step 4**: Interpret the results
+
+Once the workflow has finished, we can evaluate the linear plot generated by [**`Genomescope`**](https://github.com/schatzlab/`Genomescope`) (Fig. 7), which includes valuable information such as the observed *k*-mer profile, fitted models and estimated parameters. This file corresponds to the dataset `15` in this [history](https://usegalaxy.org/u/cartman/h/k-mer-profiling).
 
-Once the workflow has finished, we can evaluate the linear plot generated by **Genomescope** (fig. 3), which includes valuable information such as the observed k-mer profile, fitted models and estimated parameters. This file corresponds to the dataset `26`.
+-----
 
-![Figure 3: Genomescope plot](../../images/vgp_assembly/genomescope_plot.png "GenomeScope2 k-mer profile. The first peak located at about 25x corresponds to the heterozygous peak. The second peak at 50x, corresponds to the homozygous peak. The plot also includes information about the the inferred total genome length (len), genome unique length percent (uniq), overall heterozygosity rate (ab), mean k-mer coverage for heterozygous bases (kcov), read error rate (err), average rate of read duplications (dup) and k-mer size (k).")
+![Figure 6: `Genomescope` plot](../../images/vgp_assembly/genomescope_plot.png "GenomeScope2 <i>k</i>-mer profile. The first peak located at about 25&times; corresponds to the heterozygous peak. The second peak at 50&times;, corresponds to the homozygous peak. The plot also includes information about the the inferred total genome length (len), genome unique length percent (uniq), overall heterozygosity rate (ab), mean <i>k</i>-mer coverage for heterozygous bases (kcov), read error rate (err), average rate of read duplications (dup) and <i>k</i>-mer size (k).")
```

Comment:
Note here that your figure caption (useful for sighted and non-sighted users) is actually quite good. You alt-text could probably be changed to say this:

```suggestion
![figured described in caption](../../images/vgp_assembly/genomescope_plot.png "GenomeScope2 <i>k</i>-mer profile. The first peak located at about 25&times; corresponds to the heterozygous peak. The second peak at 50&times;, corresponds to the homozygous peak. The plot also includes information about the the inferred total genome length (len), genome unique length percent (uniq), overall heterozygosity rate (ab), mean <i>k</i>-mer coverage for heterozygous bases (kcov), read error rate (err), average rate of read duplications (dup) and <i>k</i>-mer size (k).")
```

we don't include figure numbers in alt-text since that's just noise for non-sighted users.

Reply:
is there a way to refer to a particular figure then (e.g., latex style)?

Reply:
yes you can, it is described how to do it [in the contributing tutorial here](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#adding-images-with-captions)

Reply:
To clarify: figures are always automatically numbered, you can use that number to refer to the image, (or use the method in my link above to link to it). 

The "Figure 6" in this case was only the alt-text for the image, not a caption and thus not shown anywhere, except to people using screenreaders (in which case it's not that useful)